discussment-examples/article/src/main/java/org/danekja/discussment/article/core/service/DashboardService.java

```diff
@@ -0,0 +1,30 @@
+package org.danekja.discussment.article.core.service;
+
+import org.danekja.discussment.article.core.domain.User;
+import org.danekja.discussment.core.domain.Category;
+import org.danekja.discussment.core.domain.Discussion;
+import org.danekja.discussment.core.domain.Topic;
+
+public interface DashboardService {
```

Comment:
I presume this is meant for "permission management", so that it is possible to show which permissions each user has.

Rename this interface to AccessControlManagerService and move it to core to the same package where is AccessControlService. Make the DefaultPermissionService implement this interface so that we can reuse the code for loading permissions and checking them.

Update documentation to describe purpose of this new AccessControlManagerService.